cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -0,0 +1,75 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2014-2015 Cask Data, Inc.
+
+.. _preferences:
+
+=================================
+Preferences and Runtime Arguments
+=================================
+
+.. include:: ../../../_common/_include/include-v260-deprecate-procedures.rst
+
+Preferences provide the ability to save configuration information at various levels of the system, including the
+CDAP Instance, Namespace, Application, and Program levels. A configuration is represented by a map of string-string
+pairs. Preferences can be retrieved, saved, and deleted through a RESTful API and through the Command Line Interface.
+When programs are started, all the Preferences at the different levels are collapsed into a single map. Preferences
+are persisted across a restart of Programs and CDAP.
+
+The overriding order of Preferences is:
```

Comment:
For some reason, it looked -- to me -- like it made sense to use > instead of <, but I'm happy to follow your lead. But what about adding the text, just to remove any ambiguity for those of us who are backwards? 
```
The overriding order of Preferences is (from lowest to highest precedence):
```